src/App.css

```diff
@@ -22,3 +22,8 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+/*a, a:active, a:hover, a:visited, a:link {*/
+  /*text-decoration: none;*/
+  /*color: black;*/
+/*}*/
```

Comment:
There's rarely any good reason to check in commented code. If you think you're going to need it later, you can go back to source-control to see it.